ui/pages/add-nft/add-nft.js

```diff
@@ -191,13 +274,21 @@ export default function AddNft() {
               dataTestId="token-id"
               titleText={t('tokenId')}
               placeholder={t('nftTokenIdPlaceholder')}
-              value={tokenId}
+              value={checkMultiNFT ? multipleTokenIds : tokenId}
               onChange={(val) => {
                 validateAndSetTokenId(val);
                 setNftAddFailed(false);
               }}
               tooltipText={t('importNFTTokenIdToolTip')}
             />
+            <Text variant={TextVariant.bodySm}>Add multiple NFT</Text>
```

Comment:
![Screen Shot 2023-04-18 at 2 43 33 PM](https://user-images.githubusercontent.com/20778143/232879887-229e0ef2-b36d-47ae-ad72-9aafbe91be38.png)

padding and alignment should be updated